src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSpec.scala

```diff
@@ -201,25 +201,18 @@ class SubmissionSpec(_system: ActorSystem) extends TestKit(_system) with FlatSpe
     checkSubmissionStatus(workspaceService, data.submissionId)
   }
 
-  it should "return a successful Submission when given an entity expression that evaluates to an empty set of entites" in withWorkspaceService { workspaceService =>
+  it should "404 when given an entity expression that evaluates to an empty set of entities" in withWorkspaceService { workspaceService =>
```

Comment:
I think I need to wait for consensus on this one.  Should we also change the (arguably) parallel case where there is no expression and we're supplying a single Entity to be consistent?